guide/spanish/cplusplus/arrays/index.md

```diff
@@ -2,41 +2,38 @@
 title: C++ Arrays
 localeTitle: Arrays C ++
 ---
-## ¿Qué son las matrices?
+## ¿Qué son los arreglos?
 
-Una matriz es una serie de elementos del mismo tipo de datos que se almacenan en ubicaciones de memoria contiguas y pueden referenciarse individualmente.
+Un arreglo es una serie de elementos del mismo tipo de datos que se almacenan en ubicaciones de memoria contiguas y pueden referenciarse individualmente.
 
-Por ejemplo, una matriz que contiene 5 valores enteros llamados números se declara así:
+Por ejemplo, un arreglo que contiene 5 valores enteros llamados **numeros** se declara así:
 
 ```C++
-int numbers [5]; 
+int numeros[5]; 
 ```
 
 Inicialización:
 
 ```C++
-//Initialization with entries: 
- int numbers [5] = {1, 2, 3, 4, 5}; 
+ //Inicializando el arreglo con valores previos
+ int numeros[] = {1, 2, 3, 4, 5}; 
  
- //Initialization with no values: 
- int numbers [5] = {}; 
+ //Inicializando el arreglo sin datos
+ int numeros[5]; 
```

Comment:
While the effect is the came in c++ (invalid in c), I think it'd be better to leave the explicit initialization in.